vcpkg/examples/snippets/get-started-linux/main.cpp

```diff
@@ -0,0 +1,7 @@
+#include <fmt/core.h>
+
+    int main()
+    {
+    fmt::print("Hello World!\n");
```

Comment:
Why isn't code indented? It lines up with the brace. That doesn't match any of our code formatting in Learn.